.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: Build & Tests (server)
+
+on:
+  push:
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build_test_run:
+    name: Build and test
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: x86_64-unknown-linux-gnu
+
+      - name: Use cached dependencies
+        uses: Swatinem/rust-cache@v2
```

Comment:
Same question as above.

Reply:
I don't see you using a cache in that workflow? Swatinem's version is again actively maintained, I even talked to him once to resolve some issue I had

Reply:
Great, also good to know.